gateway-shell/src/main/java/org/apache/knox/gateway/shell/KnoxSession.java

```diff
@@ -593,7 +593,26 @@ public String toString() {
     String s = JsonUtils.renderAsJsonString(map);
     String home = System.getProperty("user.home");
     try {
-      write(new File(home + File.separator + ".knoxshell" + File.separator + fileName), s);
+      write(new File(
+          home + File.separator + ".knoxshell" + File.separator + fileName), s);
```

Comment:
Just a note for future reference:

This could be `Paths.get(home, ".knoxshell", fileName).toFile()` or just use the `Path` directly instead of manually constructing a path.

Reply:
Nice. Thanks.